grafana/docker-build/dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM grafana/grafana
+FROM grafana/grafana:6.3.6
```

Comment:
Did you make sure that all stays working after this changes?

Reply:
No but made sure this is the version with `ubuntu` in it. The new `latest` image(which is default) now based on `Apline` and since I ain't sure of purpose of this change, I kept it all as is.

Reply:
I think the original author can fix it if necessary.

Reply:
But it is a bad practice not to set the version if you're inherit from another image.